docs/en-us/sidecarset.md

```diff
@@ -278,6 +278,72 @@ spec:
 ```
 **Note: If you use Scatter, it is recommended to set only a pair of key-values for scatter. It will be easier to understand.**
 
+### Hot Upgrade Sidecar
+Sidecar's in-place upgrade method is to stop the old version of the container first and then create the new version of the container, this upgrade method is more suitable for stateless sidecar containers, e.g. LogTail.
```

Comment:
SidecarSet's in-place upgrade will stop the container of old version first and then create the container of new version. 
Such method is more suitable for stateless sidecar containers, e.g.  logging collector